### ISSUE TITLE
historicengland.org.uk - "Aerial Photograph EXplorer"

### ISSUE THREAD

**Issue by pigsonthewing** [lovasoa/dezoomify#619]
Hello everyone,

 I am having issues when trying to download https://historicengland.org.uk/images-books/archive/collections/aerial-photos/record/RAF_543_1311_1F22_0312 - a new "Aerial Photograph EXplorer" launched this week, with many out-of-copyright RAF images (UK Crown copyright expires after 50 years).

Dezoomify reports:

```
Error: Unable to find a proper dezoomer for:
https://historicengland.org.uk/images-books/archive/collections/aerial-photos/record/RAF_543_1311_1F22_0312

(https://dezoomify.ophir.dev/dezoomers/automatic.js:30)
```
Can someone please assist?

**Comment by pigsonthewing** [comment]
To clarify; the new "EXplorer", which links to pages such as the above,  is at https://historicengland.maps.arcgis.com/apps/webappviewer/index.html

**Comment by UnevenCamber** [comment]
Same here, any luck downloading these?

**Comment by pigsonthewing** [comment]
Partial success reported here:

https://twitter.com/MonsieurBraun/status/1514872296367874054

The final image size seems to be dependent on the user's screen/ window size (more research needed).

**Comment by pigsonthewing** [comment]
Can someone merge this with https://github.com/lovasoa/dezoomify/issues/622 please?

**Comment by pigsonthewing** [comment]
This technology is also being used by Tate; example: https://www.tate-images.com/preview.asp?item=N01547&itemw=200&itemf=0001&itemstep=351&itemx=388

**Comment by lovasoa** [comment]
Closed in favor of #622

**Comment by lovasoa** [comment]
The solution in #622 works for historicengland:

![index](https://user-images.githubusercontent.com/552629/182023034-823369fb-5067-4346-a529-7cba8b14343d.jpeg)
